core-web/apps/dotcms-ui/src/app/portlets/dot-experiments/dot-experiments-list/dot-experiments-list.component.ts

```diff
@@ -51,7 +58,18 @@ export class DotExperimentsListComponent implements OnInit {
      * @memberof DotExperimentsListComponent
      */
     addNewExperiment() {
-        // To implement
+        const viewContainerRef = this.dotDynamicHost.viewContainerRef;
+        viewContainerRef.clear();
+        const componentRef = viewContainerRef.createComponent<DotExperimentsCreateComponent>(
+            DotExperimentsCreateComponent
+        );
+        componentRef.instance.vm$
+            .pipe(
+                takeUntil(this.unsubscribe$),
+                filter(({ isOpenSidebar }) => !isOpenSidebar),
+                delay(500)
+            )
+            .subscribe(() => viewContainerRef.clear());
```

Comment:
Can you just emit an event from the component and capture it here? this is a lot.